File: @LV1/1.5/onechicken/onechicken.py
a, b = [int(x) for x in input().split(' ')]
diff = a - b

p = "piece"
if abs(diff) > 1:
    p += 's'

if diff > 0:
    print(f"Dr. Chaz needs {diff} more {p} of chicken!")
else:
    print(f"Dr. Chaz will have {-diff} {p} of chicken left over!")

File: @LV1/1.5/onechicken/onechicken.hs
main :: IO ()
main = do
  [a, b] <- map read . words <$> getLine
  let diff = a - b
      p    = if abs diff /= 1 then "pieces" else "piece"

  putStrLn $ if diff > 0
    then "Dr. Chaz needs " ++ show diff ++ " more " ++ p ++ " of chicken!"
    else "Dr. Chaz will have " ++ show (-diff) ++ " " ++ p ++ " of chicken left over!"
